cron/config/config.go

```diff
@@ -147,6 +147,17 @@ func GetBigQueryTable() (string, error) {
 	return getStringConfigValue(bigqueryTable, configYAML, "BigQueryTable", "bigquery-table")
 }
 
+// UPGRADEv2: to remove.
+// GetBigQueryTable2 returns the table name to transfer cron job results.
+func GetBigQueryTable2() (string, error) {
```

Comment:
You'll need to update the `config` struct to add `BigQueryTableV2` and `ResultDataBucketURLV2` fields. Also, need to defined `env` vars that correspond to these.

Reply:
done. ptal